### ISSUE TITLE
[a11y] 9.1.3.1g - No structure markup for layout tables

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5381]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

Only note: Layout table "Versionen": For layout tables the ARIA role="presentation" should be used.

**relates:**
- Page 1 (Alle Dateien)
- Page 2 (Mit anderen geteilten Dateien)

## Status: fulfilled

**Comment by pascalwengerter** [comment]
Versions now uses lists (of lists), so not applicable any more
